setup.cfg

```diff
@@ -45,7 +45,7 @@ install_requires =
 
 [options.extras_require]
 channels =
-    Django>=2.2, <2.3
-    channels==1.1.8
-    asgi_redis
+    Django>=2.2, <3.2
```

Comment:
why not 3.2?

Reply:
Good question, I might have push this change this by mistake while trying to solve the version conflict. 

In requirements.txt there is a restriction on django version though (< 3.2), should we remove the restriction there as well ?

Reply:
we absolutely should.